src/main/java/com/puppycrawl/tools/checkstyle/checks/regexp/RegexpMultilineCheck.java

```diff
@@ -73,6 +73,20 @@
  * </li>
  * </ul>
  * <p>
+ * To run the check with its default configuration:
+ * </p>
+ * <pre>
+ * &lt;module name=&quot;RegexpMultiline&quot;&gt;
+ * &lt;/module&gt;
```

Comment:
use single line format for default config
example - https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/new_branch_2021232544/config_annotation.html#AnnotationLocation_Examples

Reply:
@stryder2000 Please do not mark the conversation as resolved yourself. It is the reviewers who mark it, so that nothing get missed.

If you have done your changes, just comment a "Done", and the reviewer could get the idea of your changes.

Reply:
Here change
```
 * &lt;module name=&quot;RegexpMultiline&quot;&gt;
 * &lt;/module&gt;
```
to
```
 * &lt;module name=&quot;RegexpMultiline&quot; /&gt;
```
like this: 
![image](https://user-images.githubusercontent.com/48255244/107221452-ac388f00-6a39-11eb-800e-429ab24d6018.png)

Reply:
> If you have done your changes, just comment a "Done", and the reviewer could get the idea of your changes.
Okay,